module/support/logging/LegLoadsLogger/src/LegLoadsLogger.cpp

```diff
@@ -23,22 +23,20 @@ namespace support {
             : Reactor(std::move(environment)), logFile(), logFilePath("NON_EXISTENT_FILE.CSV") {
 
             on<Configuration>("LegLoadsLogger.yaml")
-                .then("Leg Loads Logger Configuration", [this](const Configuration& config) {
-                    // logFilePath = config["log_file"].as<std::string>("NON_EXISTENT_FILE.CSV");
-                    logFilePath = config["log_file"].as<std::string>();
-                });
+                .then("Leg Loads Logger Configuration",
```

Comment:
It would be good if a future PR could get rid of this and just use the datalogger module

Reply:
Yea, a future PR